features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/TreeViewTraversal.kt

```diff
@@ -76,6 +81,14 @@ internal class TreeViewTraversal(
             )
         }
 
+        val resolvedWireframes = internalLogger.measureMethodCallPerf(
+            javaClass,
```

Comment:
Similar to what was done with WindowsOnDrawListener, do we need to also add this class to proguard rules to prevent classname obfuscation?

Reply:
Makes sense indeeed